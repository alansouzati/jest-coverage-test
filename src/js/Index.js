// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';

var hi = "hi";

export default class Index extends React.Component {

  _onClickTest () {
    console.log('hi');
  }

  render () {
    return <div onClick={this._onClickTest}>{this.props.children}</div>;
  }
}
