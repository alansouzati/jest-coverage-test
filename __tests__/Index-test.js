import React from 'react';
import renderer from 'react/lib/ReactTestRenderer';

import Index from '../src/js/Index';

describe('Index', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Index />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
