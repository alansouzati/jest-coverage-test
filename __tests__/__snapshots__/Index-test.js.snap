exports[`Index has correct default options 1`] = `
<div
  onClick={[Function _onClickTest]} />
`;
